packages/augur-sdk/src/api/ZeroX.ts

```diff
@@ -521,14 +521,18 @@ export class ZeroX {
     try {
       if (this._mesh) {
         return this._mesh.addOrdersAsync(orders);
-      } else {
+      } else if (this._rpc) {
         return this._rpc.addOrdersAsync(orders);
+      } else {
+        throw Error('0x rpc and browser mesh are undefined so orders cannot be created')
```

Comment:
This error message ends up being shown to the user so suggestions for better copy are appreciated.

Reply:
Can we add something about *why* they are undefined?